clients/go/admin/pkce/token_cache_inmemory.go

```diff
@@ -0,0 +1,24 @@
+package pkce
+
+import (
+	"fmt"
+
+	"golang.org/x/oauth2"
+)
+
+type TokenCacheInMemoryProvider struct {
+	token *oauth2.Token
+}
+
+func (t *TokenCacheInMemoryProvider) SaveToken(token *oauth2.Token) error {
```

Comment:
if this is the only impl, why even have an error as a response?

Reply:
The other implementation is in flytectl... it's pluggable when building the Clientset (see NewClientSet().WithTokenCache)

Reply:
oh, thanks for explaining!